extensions/functions_aggregate_generic.yaml

```diff
@@ -35,3 +35,41 @@ aggregate_functions:
             value: any
         nullability: DECLARED_OUTPUT
         return: any?
+  - name: "first"
+    description: >-
+      First value from a group of values.
```

Comment:
It's probably worth noting that order matters for these two functions.  Acero will reject plans that don't have a defined ordering on the input which might be a reasonable practice.

Reply:
Added a note c93c326bfd5d41c94825a0dffd2cde2089e6fce0!